### DISCUSSION TITLE
Dump of the links to ythlsgo.onrender.com

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#529] [Show and tell]
Link to download: https://freenote.biz/D1qmkPDBmX

**Comment by CasperMcFadden95** [comment]
VLC & MPV have YouTube support.

I think we should keep them in the streams/*.m3u like this:
`https://youtube.com/channel/UCV01ajGl6nt09h40iDoHDNg/live`

And then update the scripts so the default playlist doesn't include them and we have another generated playlist with them?

**Reply by BellezaEmporium** [reply]
Doing this restrains us in giving an universal solution.
